.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,39 +1,48 @@
 ---
 name: Bug Report
 about: Template to report bugs
-title: ''
+title: 'bug: '
 labels: ''
 assignees: ''
 
 ---
 
-Description of the problem:
+# Bug Report
+
+## Capacitor Version
+<!-- Provide the version of Capacitor you are using in your application. This can be found in your package.json file. -->
+
+## Affected Platform(s)
```

Comment:
Maybe add the checkbox comment you have in the feature request template, and tell them to leave empty and check after creating the issue, I've seen a lot of people struggle with this and making invalid checkboxes or deleting the options they don't want